src/pybind/mgr/progress/module.py

```diff
@@ -2,7 +2,7 @@
 import threading
 import datetime
 import uuid
-
+import time
```

Comment:
note: in master, this import was added by https://github.com/ceph/ceph/pull/28907 - should we consider backporting that to nautilus?

Reply:
Yes, it is now required